### PR TITLE
ZNTA-2134 add doc for version TM merge

### PR DESCRIPTION
**Pull request by huangp**
JIRA issue URL: https://zanata.atlassian.net/browse/ZNTA-2134

## Reviewer tasks

The following is based on the
[Zanata Development Guidelines](https://github.com/zanata/zanata-platform/wiki/Development-Guidelines).
As a reviewer, you should check the guidelines regularly for updates, and just
to refresh your memory.

The reviewer can use this list for reference to make sure
they have considered all the important points.

Use the checkboxes or not, whatever works best for you.

 - Code quality
   - [ ] Code passes style check (checkstyle/eslint)
   - [ ] Code is clear and concise
   - [ ] UI code is internationalised
   - [ ] Code has adequate comments where appropriate
   - Code is in an appropriate place
     - [ ] Classes are in appropriate packages
     - [ ] Code fits with class's responsibilities (SRP)
   - [ ] Configuration files are documented enough
   - If code is removed
     - [ ] No remaining code references the removed code
       - [ ] No orphaned rewrite rules
       - [ ] No orphaned config settings
     - [ ] No remaining comments refer to the removed code
     - [ ] Unused imports and libraries are removed
 - Testing
   - [ ] New code has tests
   - [ ] Changed code has tests
   - [ ] All tests pass
   - [ ] Test coverage stable or increased
 - Documentation
   - [ ] New features are documented
   - [ ] Docs removed for deleted features
   - [ ] Docs updated for all changed features
   - [ ] Developer/sysadmin docs updated if appropriate
 - If there are new dependencies
   - [ ] Does each new dependency pass all the
         [Considerations for new dependencies](https://github.com/zanata/zanata-platform/wiki/Development-Guidelines#new-technologies-and-dependencies)?

----
*This template can be updated in .github/PULL_REQUEST_TEMPLATE.md*

---
This change is [Reviewable](https://reviewable.io/reviews/zanata/zanata-platform/456)

### CODE REVIEW

File: docs/user-guide/versions/version-tm-merge.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        <td>From imported TM (TMX import)</td><td>Don't copy</td><td>No</td>
+    </tr>
+    <tr>
+        <td>From imported TM (TMX import)</td><td>Copy as fuzzy</td><td>Saved as fuzzy</td>
+    </tr>
+    <tr>
+        <td>From imported TM (TMX import)</td><td>Copy as fuzzy</td><td>Saved as fuzzy</td>
+    </tr>
+    <tr>
+        <td>TM origin</td><td>From any project version or from selected project versions</td><td></td>
```
Comment (djansen-redhat) [inline docs/user-guide/versions/version-tm-merge.md]: Not sure what this means - is there and outcome?

File: docs/user-guide/versions/version-tm-merge.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        <td>Different msgctxt or res ID (message context)</td><td>Copy as fuzzy</td><td>Saved as fuzzy</td>
+    </tr>
+    <tr>
+        <td>From imported TM (TMX import)</td><td>Don't copy</td><td>No</td>
+    </tr>
+    <tr>
+        <td>From imported TM (TMX import)</td><td>Copy as fuzzy</td><td>Saved as fuzzy</td>
+    </tr>
+    <tr>
+        <td>From imported TM (TMX import)</td><td>Copy as fuzzy</td><td>Saved as fuzzy</td>
```
Comment (djansen-redhat) [inline docs/user-guide/versions/version-tm-merge.md]: This line is a duplicate I think

**Comment by huangp** [comment]
Review status: 0 of 4 files reviewed at latest revision, 4 unresolved discussions.

---

*[docs/user-guide/versions/version-tm-merge.md, line 1 at r1](https://reviewable.io:443/reviews/zanata/zanata-platform/456#-KqQGQsmTezih2Hr3bqk-r1-1:-KqR2yZOJAiU-bUdV-K1:b-896fix) ([raw file](https://github.com/zanata/zanata-platform/blob/f52606fb4573c09633b88ae4f934a1078d066a6f/docs/user-guide/versions/version-tm-merge.md#L1)):*
<details><summary><i>Previously, djansen-redhat (Damian Jansen) wrote…</i></summary><blockquote>

Merge translations from Translation Memory
</blockquote></details>

Done.

---

*[docs/user-guide/versions/version-tm-merge.md, line 11 at r1](https://reviewable.io:443/reviews/zanata/zanata-platform/456#-KqQGQsmTezih2Hr3bqk-r1-11:-KqR2z8JVC7plfE43Rg3:b-896fix) ([raw file](https://github.com/zanata/zanata-platform/blob/f52606fb4573c09633b88ae4f934a1078d066a6f/docs/user-guide/versions/version-tm-merge.md#L11)):*
<details><summary><i>Previously, djansen-redhat (Damian Jansen) wrote…</i></summary><blockquote>

'over' is unnecessary
</blockquote></details>

Done.

---

*[docs/user-guide/versions/version-tm-merge.md, line 38 at r1](https://reviewable.io:443/reviews/zanata/zanata-platform/456#-KqQGQsmTezih2Hr3bqk-r1-38:-KqR2zokaxyOE7t9mVV3:b-896fix) ([raw file](https://github.com/zanata/zanata-platform/blob/f52606fb4573c09633b88ae4f934a1078d066a6f/docs/user-guide/versions/version-tm-merge.md#L38)):*
<details><summary><i>Previously, djansen-redhat (Damian Jansen) wrote…</i></summary><blockquote>

This line is a duplicate I think
</blockquote></details>

Done.

---

*[docs/user-guide/versions/version-tm-merge.md, line 41 at r1](https://reviewable.io:443/reviews/zanata/zanata-platform/456#-KqQGQsmTezih2Hr3bqk-r1-41:-KqR3-KdYcTu1lBmAfVB:b-2alajx) ([raw file](https://github.com/zanata/zanata-platform/blob/f52606fb4573c09633b88ae4f934a1078d066a6f/docs/user-guide/versions/version-tm-merge.md#L41)):*
<details><summary><i>Previously, djansen-redhat (Damian Jansen) wrote…</i></summary><blockquote>

Not sure what this means - is there and outcome?
</blockquote></details>

I added an outcome.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/zanata/zanata-platform/456)*

**Comment by huangp** [comment]
Cherry picked the change already.

**Pull request closed without merging**